futures-util/src/async_await/pending.rs

```diff
@@ -3,6 +3,7 @@ use futures_core::future::Future;
 use futures_core::task::{self, Poll};
 
 /// A macro which yields to the event loop once.
+/// 
```

Comment:
Replace the last paragraph with ``"This is equivalent to returning [`Poll::Pending`] from a [`Future::poll`] implementation. Similarly, when using this macro, it must be ensured that [`wake`](std::task::Waker::wake) is called somewhere when further progress can be made."``

(I rephrased it a bit an added links)